plugins/lookup/revbitspss.py

```diff
@@ -0,0 +1,41 @@
+from ansible.errors import AnsibleError, AnsibleOptionsError
+
+sdk_is_missing = False
+try:
+    from pam.secrets.server import (
+      SecretServer,
+      SecretServerError,
+    )
+except ImportError:
+    sdk_is_missing = True
+from ansible.utils.display import Display
+from ansible.plugins.lookup import LookupBase
+
+
+display = Display()
+
+
+class LookupModule(LookupBase):
+    @staticmethod
+    def Client(server_parameters):
+        return SecretServer(**server_parameters)
+
+    def run(self, terms, variables, **kwargs):
+        if sdk_is_missing:
+            raise AnsibleError("python-pss-sdk must be installed to use this plugin")
```

Comment:
https://pypi.org/project/python-pss-sdk/ doesn't exist. Also, please check https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/dev_guide/testing/sanity/import.rst on how to raise this exception correctly.